client/signup/steps/domains/index.jsx

```diff
@@ -653,6 +653,25 @@ class DomainsStep extends Component {
 			);
 		}
 
+		if ( HOSTING_LP_FLOW === flowName ) {
+			const components = {
+				span: (
+					// eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/interactive-supports-focus
+					<span
+						role="button"
+						className="tailored-flow-subtitle__cta-text"
+						style={ { fontWeight: 500 } }
+						onClick={ () => this.handleSkip( undefined, true ) }
+					/>
```

Comment:
We can just address the a11y issues by using a button

```suggestion
					<button
						className="tailored-flow-subtitle__cta-text"
						style={ { fontWeight: 500, fontSize: '1em', display: 'inline' } }
						onClick={ () => this.handleSkip( undefined, true ) }
					/>
```